src/ProjectTemplates/test/Templates.Blazor.Tests/BlazorWebTemplateTest.cs

```diff
@@ -111,5 +121,6 @@ private class BlazorTemplatePages
     {
         internal static readonly string Index = "";
         internal static readonly string FetchData = "showdata";
```

Comment:
@surayya-MS Does this need to change to "weather"?

Reply:
Yes, it does but in another PR